### PR TITLE
Fix international tests

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Add the issue ID in the customer.js file, delete a '/' in moveFile.js and add a global variable 'SHOPPATH' to add the shop path when you run the test outside your PrestaShop installation.   
| Type?         | refacto 
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH=full/11_international/* npm run specific-test -- --URL=shopURL --SHOPPATH=shopPATH

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13773)

### CODE REVIEW

File: tests/E2E/test/selectors/BO/international/translations.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = {
     translations_type:'//*[@id="form_modify_translations_translation_type"]',
     translations_language:'//*[@id="form_modify_translations_language"]',
     modify_button:'//*[@id="main-div"]//form[@name="modify_translations"]//button[@class="btn btn-primary"]/span',
-    shop_button:'(//div[@label="Shop"])[31]',
```
Comment (mickaelandrieu) [inline tests/E2E/test/selectors/BO/international/translations.js:8]: it's a super risky and error prone selector: can't we do better? :(

**Review by mickaelandrieu — DISMISSED** [review]
let's try to use a more reliable selector

File: tests/E2E/test/selectors/BO/international/translations.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/E2E/test/selectors/BO/international/translations.js:8]: Thank you for your review! 
Yes it's risky, I changed it by =>  //*[@id="app"]//li[2]/div[@label="Shop"]

Comment (mbadrani) [reply tests/E2E/test/selectors/BO/international/translations.js:8]: @mickaelandrieu do you accept the change request?

**Comment by mbadrani** [comment]
@eternoendless we need this PR can you review/merge it please?

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
@boubkerbribri do you want to have a look at this one as well?

**Comment by boubkerbribri** [comment]
@Quetzacoalt91 For me, This changes will improve tests. So no other check needed.

**Comment by Quetzacoalt91** [comment]
Thank you all!

**Pull request merged**